### ISSUE TITLE
Next Note / Previous Note commands in Excel open the native notes (rather than NVDA's overridden notes) and are not accessible

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#18713]
### Is your feature request related to a problem? Please describe.

NVDA overrides the shift+f2 command to create a note in Excel as the native note is not accessible.

The review ribbon (alt+r) has commands to jump to next note (t then t) or previous note (t then p).  These commands jump to the cell with the next (or previous) note, but then open the native note which is not accessible.

### Describe the solution you'd like

NVDA should override these commands to open it's own Note interface with the existing note for editing.

### Describe alternatives you've considered

The elements list can be used (and is the recommended way) to move to notes on the worksheet, although if users do access the ribbon commands, NVDA will stop speaking in the note and this may cause confusion.

### Additional context
What Excel now calls "Notes", used to be called "Comments". Notes are essentially a single annotation attached to a cell, and any cell can have one of these. Comments on the other hand, are more like comments in Word - they have your name against them and other users with access to the workbook can reply to comments. Press SHIFT+F2 to create a note. Press CONTROL+SHIFT+F2 to create a comment.

**Comment by Qchristensen** [comment]
I should add, this is using the default iAccessible2 Interface in Excel.  Using UIA to access Excel, the native note interface CAN be read.
